sync/class.jetpack-sync-defaults.php

```diff
@@ -104,6 +104,8 @@ class Jetpack_Sync_Defaults {
 		'users_can_register',
 		'active_plugins',
 		'uninstall_plugins',
+		'advanced_seo_front_page_description',
+		'advanced_seo_title_formats',
 	);
```

Comment:
It would be great to add `Advanced_SEO_Posts::DESCRIPTION_META_KEY` to the post_meta_whitelist array() 
see https://github.com/Automattic/jetpack/blob/master/sync/class.jetpack-sync-defaults.php#L198